packages/scripts/config/jest-environment-puppeteer/index.js

```diff
@@ -43,7 +43,8 @@ const KEYS = {
 };
 
 const root = process.env.GITHUB_WORKSPACE || process.cwd();
-const ARTIFACTS_PATH = path.join( root, 'artifacts' );
+const ARTIFACTS_PATH =
+	process.env.WP_ARTIFACTS_PATH || path.join( root, 'artifacts' );
```

Comment:
Nit: How about this:

```suggestion
	path.join( root, process.env.WP_ARTIFACTS_PATH || 'artifacts' );
```

So that we can ensure the path to be an absolute path. It shouldn't matter that much though but why not 😆.

We can also change `join` to `resolve` so that we can support paths outside the root directory.

Reply:
Makes sense. I've updated per these notes and also added a `{ recursive: true }` to the directory creation to avoid potential failure if the env var is set to a subdirectory that might not exist.